UG-Samples/Connectors/Segments/Bezier/BezierSegmentShape/BezierSegmentShapeWithoutConstraints.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Added .Net6 and .Net8 csproj file and ensure the sample run in both environment.

Like this. 
![image](https://github.com/user-attachments/assets/84e9ec82-1f5b-48cd-a3de-a96bc5cf031d)